plugin/audit_log/audit_log.c

```diff
@@ -1746,7 +1748,9 @@ static struct st_mysql_audit audit_log_descriptor=
 
 static struct st_mysql_show_var audit_log_status_variables[]=
 {
-  { 0, 0, 0}
+  {"Audit_buffer_size_too_small",
+   (char*) &audit_buffer_size_too_small,
+   SHOW_LONG}
```

Comment:
```suggestion
   SHOW_LONG},
   { 0, 0, 0}
```
Keep the terminating NULL entry